plugins/outputs/bigquery/sample.conf

```diff
@@ -4,7 +4,7 @@
   credentials_file = "/path/to/service/account/key.json"
 
   ## Google Cloud Platform Project
-  project = "my-gcp-project"
+  # project = "*detect-project-id*"
```

Comment:
This suggests that `"*detect-project-id*"` is a valid value for the parameter. Is this the case? I.e. what happens if I uncomment that option as is?

Reply:
That is the string to tell bigquery library to auto-detect the project id..

Reply:
Do you think we should hide that and use empty string?

Reply:
No it's fine if you can uncomment it and it works.